woocommerce/woocommerce
# Remove check to skip testing instructions if needed in PR template and add final check to confirm that testing instructions were added.

### All Submissions:


### Changes proposed in this Pull Request:

This PR removes the checkbox to skip testing instructions if needed when creating a Pull Request and adds another one instead to check that the PR author included testing instructions, with the intention of raising awareness on the importance of testing instructions.


### How to test the changes in this Pull Request:


1. Paste the content of the `PULL_REQUEST_TEMPLATE.md` in a markdown visualizer.
2. Check that it doesn't contain a checkbox with the following text:
```
This PR is a very minor change/addition and does not require testing instructions (if checked you can ignore/remove the next section).
```
4. Check that there is a new checkbox in the `Other information` section with the following question:
```
Have you included testing instructions?
```

**Alternatively, you also can test it as follows:**

1. Fork the WooCommerce repository.
2. Commit this change to the `PULL_REQUEST_TEMPLATE.md` in trunk in your forked WooCommerce repository.
3. Again in the forked repository, create a new branch to add a dummy change (such as a small change in a README.md file, for example).
4. Push that change to remote and open the URL to create a Pull Request (there is no need to actually create the PR, just open the template to create it).
5. Check that it doesn't contain a checkbox with the following text:
```
This PR is a very minor change/addition and does not require testing instructions (if checked you can ignore/remove the next section).
```
6. Check that there is a new checkbox in the `Other information` section with the following question:
```
Have you included testing instructions?
```

Example below:

![Screenshot 2023-02-22 at 12 40 15](https://user-images.githubusercontent.com/45979455/220609854-be0f2d9c-0713-4f48-aca8-c904ff675167.png)

### Other information:


### FOR PR REVIEWER ONLY:
